src/components/DateRangePicker.re

```diff
@@ -0,0 +1,31 @@
+type value = {
+  .
+  "initDate": Js.Date.t,
+  "endDate": Js.Date.t,
+};
+
+type state = {date: array(value)};
```

Comment:
```suggestion
type state = {date: array(Js.Date.t)};
```

Reply:
aaah just saw that this can only be an array of 2 right? so we can use a tuple 
(Js.Date.t, Js.Date.t), in runtime it is an array @eliabejr